secrets/acceptance/secret_scope_test.go

```diff
@@ -24,7 +24,7 @@ func TestAzureAccKeyVaultSimple(t *testing.T) {
 	DNSName := qa.GetEnvOrSkipTest(t, "TEST_KEY_VAULT_DNS_NAME")
 
 	client := common.CommonEnvironmentClient()
-	if client.IsAzureClientSecretSet() {
+	if client.IsAzure() {
```

Comment:
The intent of this check is to check for SP, not just Azure.

Reply:
this will soon be irrelevant